tests/foreman/cli/test_satellitesync.py

```diff
@@ -1982,44 +1681,20 @@ def test_positive_export_import_cv_incremental(self):
         :id: 3c4dfafb-fabf-406e-bca8-7af1ab551135
 
         :steps:
-            1. In upstream, Export CV version contents to a directory specified
-               in settings.
-            2. In downstream, Import these copied contents from some other
-               org/satellite.
-            3. In upstream, Add new packages to the CV.
-            4. Export the CV incrementally from the last date time.
+            1. In upstream, Export CV version contents to a directory specified in settings.
+            2. In downstream, Import these copied contents from some other org/satellite.
```

Comment:
Just terminology. Upstream is the exporting satellite, downstream is the importing one. Currently we use just one satellite and different orgs for export/import. But I plan to cover the multi-sat cases too for the sake of assertion that the artifacts get moved too.

Reply:
Great, it make sense, this naming is confusing but this can be improved in future when you write this multi-sat test, and you can try using [Broker.multi_manager](https://github.com/SatelliteQE/broker/pull/215) here for this scenario